File: Back/RecuperApp.Domain/Models/Requests/PaymentReceiptRequest.cs
namespace RecuperApp.Domain.Models.Requests
{
    public class PaymentReceiptRequest
    {
        public string EmployeeName { get; set; }
        public int EmployeeId { get; set; }
        public string Date { get; set; }
        public int TotalWeight { get; set; }
        public int TotalToPay { get; set; }
        public List<PaymentReceiptDetailRequest> Products { get; set; }
    }
}

File: Front/src/app/models/Requests/PaymentReceiptRequest.ts
import { PaymentReceiptDetailRequest } from './PaymentReceiptDetailRequest';

export class PaymentReceiptRequest {
  EmployeeName: string = '';
  EmployeeId: number = -1;
  Date: string = '';
  TotalWeight: number = 0;
  TotalToPay: number = 0;
  Products: PaymentReceiptDetailRequest[] = [];
}
